commit d816b843b38f6df117dd8e729e9afac0f22e8a41
Author: Florian Weimer <fweimer@redhat.com>
Date:   Thu Nov 24 17:04:13 2022 +0100

    : hplip-scan-hpaio-include.patch
    
    scam/sane/hpaio.c: Include orblite.h for more function prototypes
    
    Otherwise the build fails with compilers which do not support
    implicit function declarations.

diff --git a/scan/sane/hpaio.c b/scan/sane/hpaio.c
index d342626822fc2190..57d1ddea32fb5816 100644
--- a/scan/sane/hpaio.c
+++ b/scan/sane/hpaio.c
@@ -50,6 +50,7 @@
 #include "sclpml.h"
 #include "escl.h"
 #include "io.h"
+#include "orblite.h"
 #include "orblitei.h"
 
 

